cli/internal/run/dry_run.go

```diff
@@ -282,30 +289,32 @@ func commandLooksLikeTurbo(command string) bool {
 
 // TODO: put this somewhere else
 type taskSummary struct {
-	TaskID          string           `json:"taskId"`
-	Task            string           `json:"task"`
-	Package         string           `json:"package"`
-	Hash            string           `json:"hash"`
-	CacheState      cache.ItemStatus `json:"cacheState"`
-	Command         string           `json:"command"`
-	Outputs         []string         `json:"outputs"`
-	ExcludedOutputs []string         `json:"excludedOutputs"`
-	LogFile         string           `json:"logFile"`
-	Dir             string           `json:"directory"`
-	Dependencies    []string         `json:"dependencies"`
-	Dependents      []string         `json:"dependents"`
+	TaskID                 string             `json:"taskId"`
+	Task                   string             `json:"task"`
+	Package                string             `json:"package"`
+	Hash                   string             `json:"hash"`
+	CacheState             cache.ItemStatus   `json:"cacheState"`
+	Command                string             `json:"command"`
+	Outputs                []string           `json:"outputs"`
+	ExcludedOutputs        []string           `json:"excludedOutputs"`
```

Comment:
`Outputs` and `ExcludedOutputs` are redundant now, and I would like to remove them at some point in the future.